Makefile

```diff
@@ -35,13 +35,31 @@ $(BIN)/%: | $(BIN) ; $(info $(M) building $(PACKAGE)…)
 		|| ret=$$?; \
 	   rm -rf $$tmp ; exit $$ret
 
+FORCE:
+
+bin/%: cmd/% FORCE
+	$(GO) build -mod=vendor $(LDFLAGS) -v -o $@ ./$<
+
 KO = $(BIN)/ko
 $(BIN)/ko: PACKAGE=github.com/google/ko/cmd/ko
 
 .PHONY: apply
 apply: | $(KO) ; $(info $(M) ko apply -f config/) @ ## Apply config to the current cluster
-	$Q $(KO) apply --strict -f config
+	$Q $(KO) apply -f config
```

Comment:
why did you remove the --strict, is that intentional?

Reply:
Yes and no :sweat_smile: With `--strict`, it doesn't build any image for me (for some weird reason).

Reply:
`--strict` should only filter out things prefixed with `ko://`, so if you are seeing stuff that is prefixed not building we should talk :)

Reply:
@mattmoor just realized why… `ko://` got temporarly remove because of https://github.com/google/ko/issues/164 I think :stuck_out_tongue_closed_eyes: hence why it doesn't build a thing :stuck_out_tongue_closed_eyes:

Reply:
maybe you could add the `strict` as an option?